libraries/AP_Torqeedo/AP_Torqeedo.h

```diff
@@ -0,0 +1,111 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+
+#ifndef TORQEEDO_ENABLED
+#define TORQEEDO_ENABLED !HAL_MINIMIZE_FEATURES
+#endif
+
+#if TORQEEDO_ENABLED
+
+#include <AP_Param/AP_Param.h>
+
+#define TORQEEDO_MESSAGE_LEN_MAX    30  // messages are no more than 30 bytes
+
+class AP_Torqeedo {
+public:
+    AP_Torqeedo();
+
+    /* Do not allow copies */
+    AP_Torqeedo(const AP_Torqeedo &other) = delete;
+    AP_Torqeedo &operator=(const AP_Torqeedo&) = delete;
```

Comment:
We have a fancy define for this.

```suggestion
    CLASS_NO_COPY(AP_Torqeedo);
```

Reply:
txs, moved to use     CLASS_NO_COPY(AP_Torqeedo);